README.md

```diff
@@ -68,4 +80,4 @@ Tests for the core python functionality to call the aws cli with the right args
 
 Tests for the shell scripts that wrap around the python script to set env vars when sourced:
 
-    ./tests/tests.sh
+    ./tests/tests.sh && ./tests/tests.fish
```

Comment:
These readme changes are all great, thanks! (Just the one ask above)

Reply:
Sounds good, addressed them and left a question and a comment